doc/tutorials/emst.md

```diff
@@ -18,7 +18,7 @@ via templates.  For more details, see the following paper:
 
 ```
 @inproceedings{march2010fast,
-  title={Fast {E}uclidean minimum spanning tree: algorithm, analysis, and
+  title={Fast Euclidean minimum spanning tree: algorithm, analysis, and
```

Comment:
Actually this is intentional---if you compile it in BibTeX, it will use a lowercase `e`.  So the `{E}` forces it to be uppercase.